pagopa/pn-frontend
# fix: removes intesa provider and adds infocamere, change background c…

## List of changes proposed in this pull request
- Removes intesa provider
- Adds infocamere
- Change background color to white for spid selection window

## How to test
Check that intesa is not present in spid selection page, but infocamere is, check that background color is white on the same page.
